students/ravi/Session03/mailroom.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+donation_db = ["Rob Royal",[1090.89,878.67,789.90],"Richie Rich",[560.02,434.05],"Ashton Average",[120.03,98.53],"Paul Poor",[10.03],"Joe Jobless",[1.03,2.04]]
+user_choice=''
+
```

Comment:
This is a really big block, which can make the logic difficult to follow. Try to keep the blocks small, by calling functions from within the while loop. If you are really stuck on how to do this, take a look at our solution, and step through the logic in your head.